app/models/user.rb

```diff
@@ -1,9 +1,9 @@
 class User < ApplicationRecord
-            # Include default devise modules.
-            devise :database_authenticatable, :registerable,
-                    :recoverable, :rememberable, :trackable, :validatable,
-                    :confirmable, :omniauthable
-            include DeviseTokenAuth::Concerns::User
+  # Include default devise modules.
+  devise :database_authenticatable, :registerable,
```

Comment:
These modifications should not be here but in the previous PR. Maybe you did something wrong during you rebase.